docs/SETUP_CI_CD_PIPELINE.md

```diff
@@ -198,7 +199,9 @@ so, there should be two versions of each variable, one for each environment.
 | DEPLOYMENT_ENV | live | x | live|
 | gigadb_db_password| 1234 | v | staging |
 | gigadb_db_password| 5678 | v | live |
-
+| PORTAINER_PASSWORD | "password for dev" | v | dev |
+| PORTAINER_PASSWORD | "password for staging" | v | staging |
+| PORTAINER_PASSWORD | "password for live" | v | live |
```

Comment:
This problem is not specific to `PORTAINER_PASSWORD` but if a Gitlab variable with the same name is created for different environments then all of them will be copied into the `.secrets` file by `generate_config.sh`  when `./up.sh` is executed. For example, in my latest `.secrets` file, there is now:
```
MAILCHIMP_TEST_EMAIL=mailchimp_gigadb_test@rijam.ml1.net
PORTAINER_PASSWORD=P*****
PORTAINER_PASSWORD=d*******
PORTAINER_PASSWORD=L******
```

I then don't know which one has been used to secure my local portainer at http://localhost:9009 - I have to test all of the 3 passwords in turn until I gain access.

@rija Maybe all of the Gitlab variables at the project level should have a `dev` environment version and we update `generate_config.sh` to only retrieve `dev` variables into `.secrets`?

Reply:
Hi @pli888 I didn't see your reply and I made the comment below which answer your question